### PR TITLE
Some fixes in docs

### PR DESCRIPTION
**Pull request by vsn4ik**
Change range example BS2 to BS3.

### CODE REVIEW

File: docs/markup.rst [hunk 1]
```diff
@@ -73,11 +77,13 @@ Example to save the embedded datepicker value to a hidden field
 .. code-block:: html
 
     <div id="datepicker" data-date="12/03/2012"></div>
-    <input type="hidden" id="my_hidden_input" />
+    <input type="hidden" id="my_hidden_input">
```
Comment (acrobat) [inline docs/markup.rst]: the end of the input tag should be />

**Comment by acrobat** [comment]
only one remark, the rest of the PR looks good!

File: docs/markup.rst (hunk 1 above)
Comment (vsn4ik) [inline docs/markup.rst]: Not for `input`. http://www.w3.org/TR/html5/syntax.html#void-elements

Comment (acrobat) [inline docs/markup.rst]: Ok!

**Comment by acrobat** [comment]
Thanks @vsn4ik!

**Pull request merged**